CHANGELOG.md

```diff
@@ -1,4 +1,7 @@
 # Changelog
+## 2.2.5 - 2021-03-12
```

Comment:
This is a breaking change. We should be bumping the major version number -> `3.0.0`.

Also, you need to update the version in `pyproject.toml` as well.